tests/LINEBot/SendTemplateTest.php

```diff
@@ -147,4 +147,51 @@ public function testPushTemplate()
         $this->assertTrue($res->isSucceeded());
         $this->assertEquals(200, $res->getJSONDecodedBody()['status']);
     }
+
+    public function testPushTemplateWithText()
+    {
+        // Support text field on postback template
+        // Ref: https://github.com/line/line-bot-sdk-php/issues/47
+        $mock = function ($testRunner, $httpMethod, $url, $data) {
+            /** @var \PHPUnit_Framework_TestCase $testRunner */
+            $testRunner->assertEquals('POST', $httpMethod);
+            $testRunner->assertEquals('https://api.line.me/v2/bot/message/push', $url);
+
+            $testRunner->assertEquals('DESTINATION', $data['to']);
+            $testRunner->assertEquals(1, count($data['messages']));
+
+            $message = $data['messages'][0];
+            $template = $message['template'];
+            $actions = $template['actions'];
+
+            $testRunner->assertEquals(3, count($actions));
+            $testRunner->assertEquals(ActionType::POSTBACK, $actions[0]['type']);
+            $testRunner->assertEquals('postback label', $actions[0]['label']);
+            $testRunner->assertEquals('post=back', $actions[0]['data']);
+            $testRunner->assertEquals('extend text', $actions[0]['text']);
+
+            return ['status' => 200];
+        };
+        $bot = new LINEBot(new DummyHttpClient($this, $mock), ['channelSecret' => 'CHANNEL-SECRET']);
+        $res = $bot->pushMessage(
+            'DESTINATION',
+            new LINEBot\MessageBuilder\TemplateMessageBuilder(
+                'alt test',
+                new ButtonTemplateBuilder(
+                    'button title',
+                    'button button',
+                    'https://example.com/thumbnail.jpg',
+                    [
+                        new PostbackTemplateActionBuilder('postback label', 'post=back', 'extend text'), // <= Test here
```

Comment:
I have a will to check backward compatibility to the old constructor signature on above.
And an added case of [this](https://github.com/line/line-bot-sdk-php/pull/51/files#diff-9605ec58e97a1527b7d1a5d5248d9bc2R185) checks the new constructor signature.
So I wrote a test case that looks duplicated at first glance.

(Am I misunderstanding about your suggestion?)